tests/integration/targets/cloudwatchevent_rule/tasks/main.yml

```
@@ -0,0 +1,286 @@
+---
+
+- module_detauls:
+  group/aws:
+    aws_access_key: "{{ aws_access_key }}"
+    aws_secret_key: "{{ aws_secret_key }}"
+    security_token: "{{ security_token | default(omit) }}"
+    region: "{{ aws_region }}"
+
+  block:
+    - name: create an ECS cluster
+      ecs_cluster:
+        name: "{{ ecs_cluster_name }}"
```

```suggestion
        name: "{{ ecs_cluster_name }}"
```
This variable is not defined anywhere.  Did you mean to use '{{ resource_prefix }}'?